echopype/convert/set_groups_ek80.py

```diff
@@ -181,6 +242,12 @@ def set_platform(self) -> xr.Dataset:
                         "standard_name": "time",
                     },
                 ),
+                "time3": (
+                    ["time3"],
+                    [self.parser_obj.environment["timestamp"]]
+                    if "timestamp" in self.parser_obj.environment
+                    else np.datetime64("NaT"),
+                ),
```

Comment:
Add attributes to time3. Use the same set of attributes as with the other time coordinates:
```python
{
    "axis": "T",
    "long_name": "Timestamps for NMEA datagrams",
    "standard_name": "time",
},
```
But change long_name to whatever is appropriate for the source of this time.